cmd/crank/xpkg/init.go

```diff
@@ -84,14 +84,14 @@ script without prompting.
 Examples:
 
   # Initialize a new Go Composition Function named function-example.
-  crossplane beta xpkg init function-example function-template-go
-
+  crossplane xpkg init function-example function-template-go
```

Comment:
Does this file also need to be moved out of this `beta` folder, so it will be included in the right place?

After checking our your branch and building, the `xpkg init` command seems to be missing for me:

```
❯ ./_output/bin/darwin_arm64/crank xpkg init
Usage: crossplane xpkg <command> [flags]

<help omitted> 

Commands:
  xpkg build      Build a new package.
  xpkg install    Install a package in a control plane.
  xpkg login      Login to the default package registry.
  xpkg logout     Logout of the default package registry.
  xpkg push       Push a package to a registry.
  xpkg update     Update a package in a control plane.

Flags:
  -h, --help       Show context-sensitive help.
      --verbose    Print verbose logging statements.

crossplane: error: unexpected argument init
```

Reply:
Ah yes good catch. I think I got mixed up because it was previously the only command under the (now deleted `beta xpkg` command. Fixed.